cloudigrade/api/authentication.py

```diff
@@ -37,19 +37,19 @@ def parse_psk_header(request):
     Get relevant information from the given request's PSK and account number headers.
 
     Returns:
-        (str, str): the tuple of psk and account_number fields.
-            If the psk header is not present and the account_number
-            header is specified, this function returns (None, None).
-            Otherwise it will return (str, str|None) with the valid
-            PSK and the account_number if specified.
+        (str, str, str): the triplet of psk, ord_id and/or account_number fields.
```

Comment:
Even if there are more than two items, it's still called a `tuple`. That's actually the class name.

```
>>> a = (1, 2, 3)
>>> type(a)
<class 'tuple'>
```